src/test/java/com/j256/ormlite/table/DatabaseTableConfigTest.java

```diff
@@ -269,13 +271,15 @@ public void testAfterFieldLoop() {
 
 	@Test
 	public void testMultiAfter() throws SQLException {
-		DatabaseTableConfig<MultipleAfterField> config =
-				DatabaseTableConfig.fromClass(databaseType, MultipleAfterField.class);
```

Comment:
Can you revert this line?  Looks like a formatting change only, right?

Reply:
Reverted the line